README.md

```diff
@@ -70,6 +71,8 @@ Do not forget to add an entry in `.travis.yml` too following other image scheme.
 
 **Volume mounting** for test configuration only need the directory name as full local path is build by the script.
 
+**Templating** is used ONLY for based image version, for library version please prefer build args. Base image version(s) should be define in a variable named `VERSION`.
```

Comment:
Enforcing best practices is always a good idea :+1:.

WDYT about :
```suggestion
**Templating** must **ONLY** be used for images with multiple base image versions. For any other variables used in a Dockerfile, prefer build args. Base image version(s) should be defined in a variable named `VERSION`.
```
With the last part removed if we don't enforce variables names ?

Reply:
Updated 👍